src/renderer/src/app.tsx

```diff
@@ -76,6 +82,31 @@ declare module '@tanstack/react-router' {
 	}
 }
 
+let sentryEnvironment: SentryEnvironment = 'development'
+
+if (__APP_TYPE__ === 'release-candidate') {
+	sentryEnvironment = 'qa'
+} else if (__APP_TYPE__ === 'production') {
+	sentryEnvironment = 'production'
+}
```

Comment:
Suggestion: pass this value from the main process, as with the other sentry values, otherwise you risk inadvertently having this as a different value in the main and renderer process, which I think would confuse things.